openpype/hosts/houdini/api/plugin.py

```diff
@@ -194,6 +194,7 @@ def create(self, subset_name, instance_data, pre_create_data):
                 self)
             self._add_instance_to_context(instance)
             imprint(instance_node, instance.data_to_store())
+            add_self_publish_button(instance_node)
```

Comment:
If this were to be merged at some point I feel like this might be something that studios want to disable / hide away and thus potentially expose this in studio settings. In the meantime however I think let's first see what everyone thinks of the PR / implementation in general.

Reply:
I will keep this one unresolved as a reminder!